components/builder-web/app/package/build-list/build-list.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="hab-build-list">
-  <ol *ngIf="builds && builds.size">
+  <ol>
```

Comment:
⚠️  This is one area where I made a small Angular change, moving the ngIf statement. 
The reason being is that I would like the table (list) headings to display even where there are not results/list items as this better indicates what will go here and is consistent with other areas of the app. 
This way the empty state text message appears below the table headings.